handler/proxy_client.go

```diff
@@ -144,10 +144,5 @@ func (p *ProxyClient) Do(req *http.Request) (*http.Response, error) {
 		return nil, err
 	}
 
-	if log.GetLevel() == log.DebugLevel && resp.StatusCode >= 400 {
-		b, _ := ioutil.ReadAll(resp.Body)
```

Comment:
Thanks for the contribution, this fix makes sense. 

But what do you think about if we just assign `resp.Body` with a new `ReadCloser` based on `b`? This way we get to keep the useful debug output at the proxy side, and the response from remote server is not swallowed too.